tests/main/interfaces-joystick/task.yaml

```diff
@@ -0,0 +1,57 @@
+summary: Ensure that the joystick interface works.
+
+details: |
+    The joystick interface allows reading and writing to joystick devices.
+
+prepare: |
+    . "$TESTSLIB/snaps.sh"
+    install_local test-snapd-joystick
+
+    . "$TESTSLIB/files.sh"
+    ensure_file_exists_backup_real /dev/input/js31
```

Comment:
What does this do?

Reply:
It creates this file which is going to be use on the test. In case the file exists, it is renamed and a new one is created. We do that because we need to modify it, so the idea is to avoid modifying the original file.